FWCore/Framework/src/defaultCmsRunServices.cc

```diff
@@ -23,7 +23,8 @@ namespace edm {
                                               "AdaptorConfig",
                                               "SiteLocalConfigService",
                                               "StatisticsSenderService",
-                                              "CondorStatusService"};
+                                              "CondorStatusService",
+                                              "XrdAdaptor::XrdStatisticsService"};
```

Comment:
I'm not overly happy about adding more default services to jobs where the framework itself does not need those services to operate.

Reply:
hi @dr15jones - perhaps you need two lists here - one for the things the FWK needs and one for the things that CMS wants by default?